### DISCUSSION TITLE
Newly created org-wide discussions

### DISCUSSION THREAD

**Discussion by ikeycode** [AerynOS/discussions#2] [Announcements]
Hey all, we're giving github discussions a trial run. Lemme know how its working for you!

**Comment by Staudey** [comment]
what is moss

**Reply by ikeycode** [reply]
Fair play for 1st. Fair damn play. xD

**Reply by ahoneybun** [reply]
![MOSS](https://github.com/user-attachments/assets/57f64b06-c6d8-4edd-a4f7-7b6c4cde4c42)

**Comment by NomadicCore** [comment]
Hello
